content/blog/metafacture-core-5.3.2/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Metafacture-core release 5.3.2
+date: "2022-05-20T15:04:03.2300Z"
+description: "This is about changes coming with the metafacture-core release 5.3.2"
+authors: [{lastname: "Christoph",
+           firstname: "Pascal"}]
+---
+
+*Preamble*
+
+This post describes the new developements in `metafacture-core release 5.3.2` since the release
+of [metafacture-core-5.3.1 in January 2022](https://blog.metafacture.org/metafacture-core-5.3.1/).
```

Comment:
```suggestion
This post describes the new developements in `metafacture-core release 5.3.2` since the [release
of metafacture-core-5.3.1](https://blog.metafacture.org/metafacture-core-5.3.1/) in January 2022.
```